e2e/config/config.go

```diff
@@ -1,26 +1,41 @@
 package config
 
 import (
+	"crypto/ecdsa"
 	"errors"
 	"fmt"
 	"os"
 
 	"github.com/btcsuite/btcd/chaincfg"
+	ethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
 	"gopkg.in/yaml.v2"
 )
 
 // Config contains the configuration for the e2e test
 type Config struct {
-	Accounts    Accounts  `yaml:"accounts"`
-	RPCs        RPCs      `yaml:"rpcs"`
-	Contracts   Contracts `yaml:"contracts"`
-	ZetaChainID string    `yaml:"zeta_chain_id"`
+	Accounts              Accounts  `yaml:"accounts"`
+	RPCs                  RPCs      `yaml:"rpcs"`
+	Contracts             Contracts `yaml:"contracts"`
+	ZetaChainID           string    `yaml:"zeta_chain_id"`
+	FungibleAdminMnemonic string    `yaml:"fungible_admin_mnemonic"`
+}
+
+// Account contains configuration for an account
+type Account struct {
+	RawAddress    string `yaml:"address"`
+	RawPrivateKey string `yaml:"private_key"`
```

Comment:
Because naming it just `Address` would be confusing as it would conflict with the `Address()` method.

I could also call it `EncodedAddress` or `AddressString` or something like that.

Or I could rename the methods `GetAddress()` or `DecodeAddress()` or `GetEVMAddress()`